### DISCUSSION TITLE
Help: Unable to Attach

### DISCUSSION THREAD

**Discussion by dadaWilliam** [srsran/srsRAN_Project#503] [General Help] [answered]
I am using Xiaomi 11, I have enabled the SA mode mentioned, and I can see the network 99901 but can not attach, why?

<img height="500" alt="image" src="https://github.com/srsran/srsRAN_Project/assets/90446055/3eb02bb6-df38-4aaf-adf5-459c5a30778c">

This is my SIM card configuration
<img width="1357" alt="image" src="https://github.com/srsran/srsRAN_Project/assets/90446055/e466a2f5-e2ea-4742-b38e-59e5904a28d4">

and this is my subscriber configuration 
<img width="942" alt="image" src="https://github.com/srsran/srsRAN_Project/assets/90446055/0e8585bd-c50c-49a1-ba79-7994ad2c1bef">
![Uploading image.png…]()

Should I enable or disable Capability and Vulnerability?
What is AMF here? What should I write? 

Any suggestion?

**Comment by brendan-mcauliffe** [comment]
Hi @dadaWilliam, it is hard to tell you what could be wrong without more information. It would help if you could share the console output, logs and pcaps with us. 

You should also take a look at the [troubleshooting section](https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html#troubleshooting) of the relevant tutorial in our documentation.

**Reply by fahad-ece** [reply]
Dear @dadaWilliam ,

Can you please share the log file? But before doing that make sure you have made necessary changes in PLMN for gNB, and amf.yaml and nrf.yaml file of core (in my case it was Open5GS). Otherwise it will not attach to the network. 

It is also important to follow the COTS UE list from srsRAN: [https://docs.srsran.com/projects/project/en/latest/knowledge_base/source/cots_ues/source/index.html](url)

For Xiaomi 11 Lite 5G NE- the PLMN of gNB must be 00101. According to the picture your IMSI PLMN is 999-01. Usually the IMSI is either 001-01 or 999-70. So make sure it is configured in a proper manner. Thank you.

**Reply by dadaWilliam** [reply]
Thank you guys, Pretty helpful!!!  I solved it by changing the PLMN and also the alfgorithm of writing my SIM card !!!

**Comment by dadaWilliam** [comment]
Thank you guys, after changing the IMSI PLMN 00101, I still can not attach 

Here is my log of amp.log:
03/12 00:53:48.357: [amf] INFO: [suci-0-001-01-0-0-0-1234567890] Unknown UE by SUCI (../src/amf/context.c:1835)
03/12 00:53:48.357: [amf] INFO: [Added] Number of AMF-UEs is now 1 (../src/amf/context.c:1616)
03/12 00:53:48.357: [gmm] INFO: Registration request (../src/amf/gmm-sm.c:1165)
03/12 00:53:48.357: [gmm] INFO: [suci-0-001-01-0-0-0-1234567890]    SUCI (../src/amf/gmm-handler.c:166)
03/12 00:53:48.497: [gmm] WARNING: Authentication failure(MAC failure) (../src/amf/gmm-sm.c:1548)
03/12 00:53:48.497: [amf] WARNING: [suci-0-001-01-0-0-0-1234567890] Authentication reject (../src/amf/nas-path.c:538)
03/12 00:53:48.638: [amf] INFO: UE Context Release [Action:3] (../src/amf/ngap-handler.c:1698)
03/12 00:53:48.638: [amf] INFO:     RAN_UE_NGAP_ID[1] AMF_UE_NGAP_ID[2] (../src/amf/ngap-handler.c:1699)
03/12 00:53:48.638: [amf] INFO:     SUCI[suci-0-001-01-0-0-0-1234567890] (../src/amf/ngap-handler.c:1702)
03/12 00:53:48.638: [amf] INFO: [Removed] Number of gNB-UEs is now 0 (../src/amf/context.c:2557)
03/12 00:53:48.638: [amf] INFO: [Removed] Number of AMF-UEs is now 0 (../src/amf/context.c:1709)
03/12 00:54:24.613: [amf] INFO: gNB-N2[127.0.0.1] connection refused!!! (../src/amf/amf-sm.c:793)
03/12 00:54:24.613: [amf] INFO: [Removed] Number of gNBs is now 0 (../src/amf/context.c:1258)

THX!!!

**Reply by dadaWilliam** [reply]
my SIM card
<img width="1105" alt="Screenshot 2024-03-11 at 6 20 13 PM" src="https://github.com/srsran/srsRAN_Project/assets/90446055/5804d570-9649-4ed6-8c89-40e6e2383307">

my subscriber:
![Screenshot from 2024-03-12 02-16-57](https://github.com/srsran/srsRAN_Project/assets/90446055/3a356861-c268-4196-9151-e963c7836323)
![Screenshot from 2024-03-12 02-17-36](https://github.com/srsran/srsRAN_Project/assets/90446055/4253667e-71c5-4711-82ea-dba2104a54c7)
![Screenshot from 2024-03-12 02-18-16](https://github.com/srsran/srsRAN_Project/assets/90446055/c90de80b-c379-4433-9bbd-c65040609729)

logs:
![Screenshot from 2024-03-12 02-18-58](https://github.com/srsran/srsRAN_Project/assets/90446055/f4bf7db3-9b7b-4ed4-9141-7bf05bd34f2d)
![Screenshot from 2024-03-12 02-19-59](https://github.com/srsran/srsRAN_Project/assets/90446055/08ec1031-54c3-4f29-8852-da182f873778)

03/12 02:10:36.346: [gmm] WARNING: [suci-0-001-01-0-0-0-1234567892] Cannot find SUCI [404] (../src/amf/gmm-sm.c:1672)
03/12 02:10:36.346: [amf] WARNING: [suci-0-001-01-0-0-0-1234567892] Registration reject [11] (../src/amf/nas-path.c:219)
03/12 02:10:36.466: [amf] INFO: UE Context Release [Action:3] (../src/amf/ngap-handler.c:1698)
03/12 02:10:36.466: [amf] INFO:     RAN_UE_NGAP_ID[8] AMF_UE_NGAP_ID[9] (../src/amf/ngap-handler.c:1699)
03/12 02:10:36.466: [amf] INFO:     SUCI[suci-0-001-01-0-0-0-1234567892] (../src/amf/ngap-handler.c:1702)
03/12 02:10:36.466: [amf] INFO: [Removed] Number of gNB-UEs is now 0 (../src/amf/context.c:2557)
03/12 02:10:36.466: [amf] INFO: [Removed] Number of AMF-UEs is now 0 (../src/amf/context.c:1709)
03/12 02:10:36.829: [amf] INFO: InitialUEMessage (../src/amf/ngap-handler.c:401)
03/12 02:10:36.829: [amf] INFO: [Added] Number of gNB-UEs is now 1 (../src/amf/context.c:2550)
03/12 02:10:36.829: [amf] INFO:     RAN_UE_NGAP_ID[9] AMF_UE_NGAP_ID[10] TAC[7] CellID[0x19b0] (../src/amf/ngap-handler.c:562)
03/12 02:10:36.829: [amf] INFO: [suci-0-001-01-0-0-0-1234567892] Unknown UE by SUCI (../src/amf/context.c:1835)
03/12 02:10:36.829: [amf] INFO: [Added] Number of AMF-UEs is now 1 (../src/amf/context.c:1616)
03/12 02:10:36.829: [gmm] INFO: Registration request (../src/amf/gmm-sm.c:1165)
03/12 02:10:36.829: [gmm] INFO: [suci-0-001-01-0-0-0-1234567892]    SUCI (../src/amf/gmm-handler.c:166)
03/12 02:10:36.831: [sbi] WARNING: [AUSF] (SCP-discover) NF has already been added [35c37a74-dfcf-41ee-b7e9-aba35b963bbf:1] (../lib/sbi/path.c:216)
 
I this something is wrong with Cannot find SUCI [404] and Unknown UE, 
Any help?

**Reply by dadaWilliam** [reply, marked as answer]
I find the answer, by changing the xor simcard with the sim card maker to milenage, it works fine !
<img width="898" alt="image" src="https://github.com/srsran/srsRAN_Project/assets/90446055/1a759292-a4a8-4b4d-9644-b9e9dbf9afa8">
You can find it here [link](https://github.com/open5gs/open5gs/issues/2726)

**Comment by dadaWilliam** [comment]
I have encountered another issue, that is, the UE's data connection can not stay stable when connected to 5G, even after using the force 5G app, when I am using the phone outside the force 5G app, it seems that the phone itself automatically changes the NR only mode to default mode and then I lose the 5G data connection and I have to go back to reset it to NR only again? It keeps happening like that, when you even just go back to the Home Screen and reopen the app, it changes!!! Why??? is this because I am using Xiaomi or MIUI? 

I am using Xiaomi 10 MIUI14 any help???

**Reply by pgawlowicz** [reply]
could you add this to your gnb config?
```
cu_cp:
  inactivity_timer: 7200            # Sets the UE/PDU Session/DRB inactivity timer to 7200 seconds. Supported: [1 - 7200].
```
